source/patterns/@aws-solutions-constructs/aws-s3-eventbridge/README.md

```diff
@@ -0,0 +1,106 @@
+# aws-s3-eventbridge module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_s3_eventbridge`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-s3-eventbridge`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.s3eventbridge`|
+
+This AWS Solutions Construct implements an Amazon S3 bucket connected to an Amazon EventBridge.
+
+Here is a minimal deployable pattern definition in Typescript:
+
+``` typescript
+import { S3ToEventbridge, S3ToEventbridgeProps } from '@aws-solutions-constructs/aws-s3-eventbridge';
+import { EventbridgeToStepfunctions, EventbridgeToStepfunctionsProps } from '@aws-solutions-constructs/aws-s3-stepfunctions';
+import * as stepfunctions from '@aws-cdk/aws-stepfunctions';
+
+const startState = new stepfunctions.Pass(stack, 'StartState');
```

Comment:
Why is there step functions content here? This construct passes S3 events to EventBridge, what happens to the event after that is out of scope.